Resources/Public/Javascript/PageView/SearchInDocument.js

```diff
@@ -56,11 +56,10 @@ $(document).ready(function() {
             "/",
             {
                 eID: "tx_dlf_search_in_document",
-                q: $( "input[name='tx_dlf[query]']" ).val(),
-                uid: $( "input[name='tx_dlf[id]']" ).val(),
-                start: $( "input[name='tx_dlf[start]']" ).val(),
-                encrypted: $( "input[name='tx_dlf[encrypted]']" ).val(),
-                hashed: $( "input[name='tx_dlf[hashed]']" ).val(),
+                q: $( "input[id='tx-dlf-search-in-document-query]']" ).val(),
```

Comment:
```suggestion
                q: $( "input[id='tx-dlf-search-in-document-query']" ).val(),
```
This doesn't work. 

I think it's a search/replace mistake.